.travis/install.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e  # exit on errors
+
+pip install --upgrade pip
+pip install -r django/requirements.txt
+pip install -r python/requirements.txt
+pip install python/
+pip install django/
+pip install https://github.com/OriHoch/Open-Knesset/archive/move-refactor-data-gathering-and-model-to-knesset-data.zip#egg=Open-Knesset
```

Comment:
before merging, need to remember to change this to point to open knesset master branch